### ISSUE TITLE
Add support for "Extract - Note (extn)" and span alt in Tor.com template

### ISSUE THREAD

**Issue by nelliemckesson** [macmillanpublishers/bookmaker_assets#41]
We need to add support for 3 new styles in the Tor.com templates (novel and novella). A new extract style, and 2 new inline spans. See below for specs. Make sure to add to the scss files! There's a readme in this repo to help guide you through the different file creation (https://github.com/macmillanpublishers/bookmaker_assets).

Designer's notes:

Both templates:
Style name: Extract - Note (extn):
centered
right and left margin: 16pt
1-line space before and after
no first-line indent

NOVEL (std)
Style name: span alt font 1 (span1)
font: Lettres Eclatees
font-size: 13.75pt
letter-spacing: .02 em

Style name: span alt font 2 (span2)
font: Berranger Hand ITC
font-size: 10.75pt
leading: 15.5
letter-spacing: 0

NOVELLA
Style name: span alt font 1 (span1)
font: Lettres Eclatees
font-size: 14pt
leading: 16pt
letter-spacing: .02 em

Style name: span alt font 2 (span2)
font: Berranger Hand ITC
font-size: 11pt
leading: 16
letter-spacing: 0

short version: in both templates, the font size for alt font 1 is 2pts larger than base with added letterspacing, and alt font 2 is 1 point smaller.

**Comment by mattretzer** [comment]
Hi @nelliemckesson, so I have this working for the pdf css... How do you want me to handle for the epub css?  I can add rules for ExtractNoteextn as per above, what do you think about the fonts/spans?

**Comment by nelliemckesson** [comment]
@mattretzer we shouldn't add the special fonts for the EPUB. EPUB font embedding is a really tricky issue, especially since Tor.com books are DRM-free. These should have the same formatting in the EPUB as the other kinds of extracts, and should be italic (can't remember if that's built into the existing extract definition).

**Comment by mattretzer** [comment]
https://github.com/macmillanpublishers/bookmaker_assets/pull/44
